pkg/agent/config/config.go

```diff
@@ -101,6 +101,14 @@ func getHostnameAndIP(info cmds.Agent) (string, string, error) {
 	return name, ip, nil
 }
 
+func getPauseImage(info cmds.Agent) (string, error) {
+	pause := info.PauseImage
+	if pause == "" {
+		return "", nil
+	}
+	return pause, nil
+}
+
```

Comment:
Thank you for submitting this PR @leo1971!

I think we can remove this function and just set `nodeConfig.AgentConfig.PauseImage = envInfo.PauseImage`

We merged in some other `config.toml` changes since this will need to be rebased.